ddl/util/util.go

```diff
@@ -145,20 +151,19 @@ func LoadDDLVars(ctx sessionctx.Context) error {
 	return LoadGlobalVars(ctx, []string{variable.TiDBDDLErrorCountLimit})
 }
 
-const loadGlobalVarsSQL = "select HIGH_PRIORITY variable_name, variable_value from mysql.global_variables where variable_name in (%s)"
-
 // LoadGlobalVars loads global variable from mysql.global_variables.
 func LoadGlobalVars(ctx sessionctx.Context, varNames []string) error {
 	if sctx, ok := ctx.(sqlexec.RestrictedSQLExecutor); ok {
-		nameList := ""
+		var buf strings.Builder
+		sqlexec.MustFormatSQL(&buf, loadGlobalVars)
 		for i, name := range varNames {
```

Comment:
You can use `%?` to output `'xx','xx'` for `[]string` type.

Reply:
addressed PTAL @xhebox

Reply:
Then these two invocation of `MustFormatSQL` can be combined into one.